contracts/pre-propose/dao-pre-propose-approver/src/contract.rs

```diff
@@ -181,6 +185,43 @@ pub fn execute_proposal_completed(
     }
 }
 
+pub fn execute_reset_approver(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+) -> Result<Response, PreProposeError> {
+    // Check that this is coming from the DAO.
+    let dao = PrePropose::default().dao.load(deps.storage)?;
+    if info.sender != dao {
+        return Err(PreProposeError::Unauthorized {});
+    }
+
+    let pre_propose_approval_contract = PRE_PROPOSE_APPROVAL_CONTRACT.load(deps.storage)?;
+
+    let reset_messages = vec![
+        // Remove the proposal submitted hook.
+        CosmosMsg::Wasm(WasmMsg::Execute {
+            contract_addr: pre_propose_approval_contract.to_string(),
+            msg: to_binary(&PreProposeApprovalExecuteMsg::RemoveProposalSubmittedHook {
```

Comment:
```suggestion
            msg: to_json_binary(&PreProposeApprovalExecuteMsg::RemoveProposalSubmittedHook {
```

`to_binary` was deprecated.